x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/page.tsx

```diff
@@ -51,6 +54,20 @@ export const Page: FC<{
     checkJobsExist();
   }, []);
 
+  useEffect(
+    function updateUrl() {
+      if (analyticsId !== undefined) {
+        setGlobalState({
+          ml: {
+            ...(analyticsId.analysis_type ? { analysisType: analyticsId.analysis_type } : {}),
```

Comment:
Is there any reason to store `analysisType` in the URL? Is it used for anything, or is the `jobId` all you need?

Reply:
This is used in the explorer page 👍